modules/jdbc-test/src/test/java/org/testcontainers/jdbc/JDBCDriverTest.java

```diff
@@ -70,6 +77,7 @@ public static void testCleanup() {
     @Test
     public void test() throws SQLException {
         performSimpleTest(jdbcUrl);
+        performContainerStatusCheck(jdbcUrl);
```

Comment:
Ok! One more thing, I see I'm getting a PMB violation about using package private method.
Would it be fine to make it `public`? I saw some classes use `VisibleForTesting` annotation, but I'm not sure if it makes any difference.